lib/requester/request-wrapper.js

```diff
@@ -115,21 +115,19 @@ var _ = require('lodash'),
         }
 
         if (!_.isString(certPath)) {
-            returncb(new Error(ERROR_MISSING_CERT_PATH));
+            return cb(new Error(ERROR_MISSING_CERT_PATH));
         }
 
-        async.mapValues([keyPath, certPath], function (value, key, cb) {
-          resolver.readFile(value, cb);
-        }, function (err, fileContents) {
+        async.map([keyPath, certPath], resolver.readFile, function (err, fileContents) {
```

Comment:
`mapValues` was correct here, you just need to call it correctly. 

1. Should pass an object to `mapValues` that looks like `{ key: keyPath, cert: certPath }`, and then access `fileContents` using `fileContents.key` and `fileContents.cert`.